command/host_create.go

```diff
@@ -137,12 +139,15 @@ func (c *createHost) waitForLogs(ctx context.Context, comm client.Communicator,
 			if status.HasStarted {
 				startedCollectingLogs = true
 				if err = c.getAndWriteLogBatch(ctx, comm, hostID, batchStart, batchEnd); err != nil {
-					return errors.Wrapf(err, "error getting and writing logs on started container")
+					logger.Task().Debugf("Tick -- error getting and writing logs on started container: '%s'", err.Error())
+					grip.Debug(message.WrapError(err, "error getting and writing logs on started container"))
```

Comment:
This will end up in a file on the host—not sure if that is your goal. The logging statement above will do the right thing though.